pallets/clusters/src/lib.rs

```diff
@@ -262,21 +262,22 @@ pub mod pallet {
 			ensure!(!settings.name.len().is_zero(), Error::<T>::InvalidInput);
 			ensure!(!settings.data.len().is_zero(), Error::<T>::InvalidInput);
 
-			let role_hash = blake2_128(&[&cluster_id[..], &role_name.to_vec().clone()[..]].concat());
+			let role_name_vec = Vec::from(role_name.clone());
```

Comment:
`as_slice`

Reply:
I already tried to use .as_slice(), but I did not use because it cause:
```rust
--> /home/workspace/fragcolor/clamor/pallets/clusters/src/lib.rs:205:38
      |
  205 |                 &[current_block_number.encode(), name.as_slice(), extrinsic_index.clone().encode(), who.clone().encode()].concat(),
      |                                                  ^^^^^^^^^^^^^^^- help: try using a conversion method: `.to_vec()`
      |                                                  |
      |                                                  expected struct `frame_support::dispatch::Vec`, found `&[u8]`
      |
      = note: expected struct `frame_support::dispatch::Vec<u8>`
              found reference `&[u8]`
```
I can use `Vec::from(name.clone().as_slice())` instead.